src/components/ColumnList.tsx

```diff
@@ -0,0 +1,21 @@
+import { WithID } from '../helpers/types'
+import { Card, Column } from '../store/store'
+import { ColumnItem } from './ColumnItem'
+
+interface Props {
+  cards: WithID<Card>[]
+  columns: WithID<Column>[]
+}
+
+export const ColumnList = ({ cards, columns }: Props) => {
+  return (
+    <>
+      {columns.map(
+        ({ title, id }) =>
+          title !== undefined && (
```

Comment:
As we discussed, you can remove that check and provide filters in `Dashboard` file. It will be easier to understand what you're doing. Because you want to filter empty documents, not documents which doesn't have `title` only.